tests/unit/EntityHeader.spec.ts

```diff
@@ -368,50 +368,63 @@ describe('Entity Header - HISTORICAL badge', () => {
   const router = mockRouter.mock()
 
   const variations = [
-    { // 0
-      entityState: 'ACTIVE',
+    { // variation 0 - active business
+      businessState: EntityState.ACTIVE,
       exists: false
     },
-    { // 1
-      entityState: 'LIQUIDATION',
+    { // variation 1 - business in liquidation
+      businessState: EntityState.LIQUIDATION,
       exists: false
     },
-    { // 2
-      entityState: 'HISTORICAL',
+    { // variation 2 - historical busines due to unknown reason
+      businessState: EntityState.HISTORICAL,
       stateFiling: null,
       exists: true,
       text: 'Unknown Reason'
     },
-    { // 3
-      entityState: 'HISTORICAL',
+    { // variation 3 - historical business due to voluntary dissolution
+      businessState: EntityState.HISTORICAL,
       stateFiling: {
-        header: { name: 'dissolution' },
+        header: { name: FilingTypes.DISSOLUTION },
         dissolution: {
           dissolutionDate: '2020-01-01',
-          dissolutionType: 'voluntary'
+          dissolutionType: FilingSubTypes.DISSOLUTION_VOLUNTARY
         }
       },
       exists: true,
       text: 'Voluntary Dissolution – January 1, 2020'
     },
-    { // 4
-      entityState: 'HISTORICAL',
+    { // variation 4 - historical company due to involuntary dissolution
+      businessState: EntityState.HISTORICAL,
       stateFiling: {
-        header: {
-          name: 'involuntaryDissolution',
```

Comment:
This was incorrect and the code was using a fallback to display the expected text.